src/Symfony/Component/DependencyInjection/Config/ContainerParametersResourceChecker.php

```diff
@@ -39,7 +39,7 @@ public function supports(ResourceInterface $metadata)
     /**
      * {@inheritdoc}
      */
-    public function isFresh(ResourceInterface $resource, $timestamp)
+    public function isFresh(ResourceInterface $resource, int $timestamp)
```

Comment:
This will break compatibility with Config 4.4. Please either revert this change or update DependencyInjection‘s composer.json accordingly.